<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogConfig</class>
 <widget class="QWidget" name="LogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messages</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="MultiColumnTreeWidget" name="qtwMessages">
     <property name="accessibleName">
      <string>Log message types and actions</string>
     </property>
     <property name="accessibleDescription">
      <string>Use up and down keys to navigate through the message types. Use left and right keys to navigate between notification possibilities for a single message type.</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Console</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Notification</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Highlight</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Text-To-Speech</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Limit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Soundfile</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgbTTS">
     <property name="title">
      <string>Text To Speech</string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="3">
       <widget class="QCheckBox" name="qcbReadBackOwn">
        <property name="toolTip">
         <string>If enabled text messages you send will be read back to you with TTS</string>
        </property>
        <property name="text">
         <string>Read back own messages</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="qlThreshold">
        <property name="text">
         <string>Length threshold</string>
        </property>
        <property name="buddy">
         <cstring>qsbThreshold</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="qcbEnableTTS">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Click to enable Text-to-Speech</string>
        </property>
        <property name="text">
         <string>Enable Text-to-Speech</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="qsbThreshold">
        <property name="toolTip">
         <string>Message length threshold for Text-To-Speech Engine</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;This is the length threshold used for the Text-To-Speech Engine.&lt;/b&gt;&lt;br /&gt;Messages longer than this limit will not be read aloud in their full length.</string>
        </property>
        <property name="accessibleName">
         <string>Set length threshold</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="suffix">
         <string> Characters</string>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="qcbNoScope">
        <property name="toolTip">
         <string>If enabled, TTS will not dictate the message scope.</string>
        </property>
        <property name="text">
         <string>Omit Message Scope</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="qcbNoAuthor">
        <property name="toolTip">
         <string>If enabled, TTS will not dictate the message author.</string>
        </property>
        <property name="text">
         <string>Omit Message Author</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgbMessageVolume">
     <property name="title">
      <string>Message Volume</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QGridLayout" name="qgbMessageVolumeGrid">
        <item row="2" column="2">
         <widget class="QSpinBox" name="qsbCueVolume">
          <property name="toolTip">
           <string>Volume adjustment for audio cues</string>
          </property>
          <property name="whatsThis">
           <string>&lt;b&gt;This is the volume adjustment for audio cues.&lt;/b&gt;&lt;br /&gt;A value of 0 dB means no change to the sound sample.</string>
          </property>
          <property name="accessibleName">
           <string>Audio cue volume adjustment</string>
          </property>
          <property name="suffix">
           <string> dB</string>
          </property>
          <property name="minimum">
           <number>-60</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="qlTTSVolume">
          <property name="text">
           <string>Text-to-Speech volume</string>
          </property>
          <property name="buddy">
           <cstring>qsTTSVolume</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="SemanticSlider" name="qsCueVolume">
          <property name="toolTip">
           <string>Volume adjustment for audio cues</string>
          </property>
          <property name="whatsThis">
           <string>&lt;b&gt;This is the volume adjustment for audio cues.&lt;/b&gt;&lt;br /&gt;A value of 0 dB means no change to the sound sample.</string>
          </property>
          <property name="accessibleName">
           <string>Audio cue volume adjustment</string>
          </property>
          <property name="minimum">
           <number>-60</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="qlNotificationVolume">
          <property name="text">
           <string>Notification volume adjustment</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="buddy">
           <cstring>qsNotificationVolume</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="qsbNotificationVolume">
          <property name="toolTip">
           <string>Volume adjustment for notification sounds</string>
          </property>
          <property name="whatsThis">
           <string>&lt;b&gt;This is the volume adjustment for notification sounds.&lt;/b&gt;&lt;br /&gt;A value of 0 dB means no change to the sound sample.</string>
          </property>
          <property name="accessibleName">
           <string>Notification sound volume adjustment</string>
          </property>
          <property name="suffix">
           <string> dB</string>
          </property>
          <property name="minimum">
           <number>-60</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SemanticSlider" name="qsNotificationVolume">
          <property name="toolTip">
           <string>Volume adjustment for notification sounds</string>
          </property>
          <property name="whatsThis">
           <string>&lt;b&gt;This is the volume adjustment for notification sounds.&lt;/b&gt;&lt;br /&gt;A value of 0 dB means no change to the sound sample.</string>
          </property>
          <property name="accessibleName">
           <string>Notification sound volume adjustment</string>
          </property>
          <property name="minimum">
           <number>-60</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SemanticSlider" name="qsTTSVolume">
          <property name="toolTip">
           <string>Volume of Text-To-Speech Engine</string>
          </property>
          <property name="whatsThis">
           <string>&lt;b&gt;This is the volume used for the speech synthesis.&lt;/b&gt;</string>
          </property>
          <property name="accessibleName">
           <string>Text to speech volume</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="qlCueVolume">
          <property name="text">
           <string>Audio cue volume adjustment</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="buddy">
           <cstring>qsCueVolume</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="qsbTTSVolume">
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgbMaxBlocks">
     <property name="title">
      <string>Chat Log</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="qcb24HourClock">
        <property name="toolTip">
         <string>If checked the time at the beginning of a message will be displayed in the 24-hour format.

The setting only applies for new messages, the already shown ones will retain the previous time format.</string>
        </property>
        <property name="text">
         <string>Use 24-hour clock</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="qsbMaxBlocks">
        <property name="accessibleName">
         <string>Maximum chat log length</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="specialValueText">
         <string>Unlimited</string>
        </property>
        <property name="suffix">
         <string> Lines</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="qlMaxBlocks">
        <property name="text">
         <string>Maximum chat length</string>
        </property>
        <property name="buddy">
         <cstring>qsbMaxBlocks</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>8</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="qlChatMessageMargins">
        <property name="toolTip">
         <string>How far individual messages are spaced out from one another.</string>
        </property>
        <property name="text">
         <string>Message margins</string>
        </property>
        <property name="buddy">
         <cstring>qsbChatMessageMargins</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="qsbChatMessageMargins">
        <property name="toolTip">
         <string>How far individual messages are spaced out from one another.</string>
        </property>
        <property name="accessibleName">
         <string>Chat message margins</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgbMisc">
     <property name="title">
      <string>Misc.</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="qcbWhisperFriends">
        <property name="toolTip">
         <string>If checked you will only hear whispers from users you added to your friend list.</string>
        </property>
        <property name="text">
         <string>Only accept whispers from friends</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Limit notifications when there are more than</string>
          </property>
          <property name="buddy">
           <cstring>qsbMessageLimitUsers</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="qsbMessageLimitUsers">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Number of users that will trigger message limiting functionality.</string>
          </property>
          <property name="accessibleName">
           <string>User limit for notifications</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>users on the server.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SemanticSlider</class>
   <extends>QSlider</extends>
   <header>widgets/SemanticSlider.h</header>
  </customwidget>
  <customwidget>
   <class>MultiColumnTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>widgets/MultiColumnTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qtwMessages</tabstop>
  <tabstop>qcbEnableTTS</tabstop>
  <tabstop>qcbNoScope</tabstop>
  <tabstop>qcbNoAuthor</tabstop>
  <tabstop>qsbThreshold</tabstop>
  <tabstop>qcbReadBackOwn</tabstop>
  <tabstop>qsTTSVolume</tabstop>
  <tabstop>qsbTTSVolume</tabstop>
  <tabstop>qsNotificationVolume</tabstop>
  <tabstop>qsbNotificationVolume</tabstop>
  <tabstop>qsCueVolume</tabstop>
  <tabstop>qsbCueVolume</tabstop>
  <tabstop>qsbMaxBlocks</tabstop>
  <tabstop>qcb24HourClock</tabstop>
  <tabstop>qsbChatMessageMargins</tabstop>
  <tabstop>qcbWhisperFriends</tabstop>
  <tabstop>qsbMessageLimitUsers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
